src/extensions/http_api/client.js

```diff
@@ -218,19 +218,25 @@ class HttpApiClient {
      * @param {string} [baseUrl] - The base url
      */
     makeRequest(method, path, baseUrl) {
+        let pathWithoutHost
```

Comment:
That's redundant, since a path does not include the host.
 ```suggestion
        let path
```